<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BaseDeDonnees</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icones.qrc">
    <normaloff>:/images/bdd.png</normaloff>:/images/bdd.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelNom">
              <property name="text">
               <string>Nom du matériau</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditNom"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelCoef">
              <property name="text">
               <string>Coefficient d'absorption</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditCoef"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelFrequence">
              <property name="text">
               <string>Fréquence</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditFrequence"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Ajouter">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset resource="icones.qrc">
            <normaloff>:/images/ajouter_ligne.png</normaloff>:/images/ajouter_ligne.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreerTableMateriau"/>
   <addaction name="separator"/>
   <addaction name="actionSupprimerTable"/>
   <addaction name="separator"/>
   <addaction name="actionRafaichir"/>
   <addaction name="separator"/>
   <addaction name="actionAfficher_l_ID"/>
   <addaction name="separator"/>
   <addaction name="actionModifier"/>
   <addaction name="separator"/>
   <addaction name="actionSupprimer"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRafaichir">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/images/rafraichir.png</normaloff>:/images/rafraichir.png</iconset>
   </property>
   <property name="text">
    <string>Rafaichir</string>
   </property>
  </action>
  <action name="actionModifier">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/images/modifier.jpg</normaloff>:/images/modifier.jpg</iconset>
   </property>
   <property name="text">
    <string>Modifier la ligne</string>
   </property>
  </action>
  <action name="actionAfficher_l_ID">
   <property name="text">
    <string>Afficher l'ID</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/images/retirer_ligne.png</normaloff>:/images/retirer_ligne.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer la ligne</string>
   </property>
  </action>
  <action name="actionCreerTableMateriau">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/images/ajouter_table.png</normaloff>:/images/ajouter_table.png</iconset>
   </property>
   <property name="text">
    <string>Créer la table</string>
   </property>
  </action>
  <action name="actionSupprimerTable">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/images/retirer_table.png</normaloff>:/images/retirer_table.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer la table</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
